site/src/components/card/Card.module.css

```diff
@@ -13,8 +13,9 @@
   --card-icon-background: var(--uitk-color-gray-10);
 }
 
-.cardTitle {
-  --ifm-heading-margin-bottom: var(--uitk-size-unit);
+.cardTitle h2 {
+  margin-top: 0;
```

Comment:
i think this aligns better set to var(--uitk-size-unit)?

Reply:
I think it's fine given the flex gap we have

Reply:
Just discussed with @libertymayc , the contributions page cards still need a couple more tweaks - let's discuss on Monday :)